xml/docu_styleguide.structure.xml

```diff
@@ -1815,7 +1817,7 @@ xml:id="tab-install-source"</screen>
      after the items that are not complete sentences. Use either all 
      full sentences in your bullet lists or all fragments. Avoid a mix.
     </para>
-    <para>Do not use commas and semicolons to end punctuation. Do not use a full stop 
+    <para>Do not use commas and semicolons to end punctuation. Do not use a period 
```

Comment:
This is not logical. In the very same bullet point, we instruct writers:

> Use a period after every list item that is a sentence. Do not use a period after the items that are not complete sentences.

So first it depends on whether it's a complete sentence or not and then we make an exception for the last item? IHMO we should just remove that sentence.

Reply:
This is about the end of punctuation for the entire list series. How about rewording it for clarity?